lib/Reader/readerir.cpp

```diff
@@ -2971,6 +2972,8 @@ FlowGraphNode *GenIR::fgSplitBlock(FlowGraphNode *Block, IRNode *Node) {
 }
 
 void GenIR::fgRemoveUnusedBlocks(FlowGraphNode *FgHead) {
+  bool IsOk = !verifyFunction(*Function, &dbgs());
+  assert(IsOk && "verification failed");
```

Comment:
Good point, added #ifndef NDEBUG.
I don't think the reader should make the assumption that we have one function in the module (and if there were more than one we wouldn't want to verify all functions every time). We still verify the module in LLILCJit::readMethod.